CMake/kwiver-depends.cmake

```diff
@@ -12,7 +12,7 @@ include( kwiver-depends-log4cxx )
 include( kwiver-depends-log4cplus )
 
 # Required for Sprokit and Track Oracle
-if(KWIVER_ENABLE_SPROKIT OR KWIVER_ENABLE_TRACK_ORACLE)
+if(KWIVER_ENABLE_SPROKIT OR KWIVER_ENABLE_TRACK_ORACLE OR KWIVER_ENABLE_TOOLS)
   include( kwiver-depends-Boost )
 endif()
```

Comment:
You can be more specific about the tools requirement for boost by using

    if ( NOT VITAL_USE_STD_REGEX )
        include( kwiver-depends-Boost )
    endif()